provisioner/node_pools.go

```diff
@@ -20,16 +22,19 @@ import (
 	"github.com/zalando-incubator/cluster-lifecycle-manager/api"
 	awsExt "github.com/zalando-incubator/cluster-lifecycle-manager/pkg/aws"
 	"github.com/zalando-incubator/cluster-lifecycle-manager/pkg/updatestrategy"
+	"gopkg.in/yaml.v2"
 )
 
 const (
 	clcFileName           = "userdata.clc.yaml"
 	cloudInitFileName     = "userdata.yaml"
 	stackFileName         = "stack.yaml"
+	pkiFileName           = "pki.yaml"
```

Comment:
Could you rename it to something more generic? We could use it for other things, not necessarily PKI.